src/unstract/sdk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.42.0"
+__version__ = "0.42.1"
```

Comment:
@harini-venkataraman since this is a new feature and not a fix, I suggest bumping the minor version
```suggestion
__version__ = "0.43.0"
```